payloads/libpayload/include/sysinfo.h

```diff
@@ -31,7 +31,7 @@
 #define _SYSINFO_H
 
 /* Maximum number of memory range definitions. */
-#define SYSINFO_MAX_MEM_RANGES 32
+#define SYSINFO_MAX_MEM_RANGES 64
```

Comment:
Why do we need to double this ranges? 32 ranges should enough for payload since it has been validated in both baremetal and emulator env.

Reply:
we checked some platforms, it will exceed 32.
sometime reach 35, sometime reach 48.

find 35 memmap ranges: 
mem[0] : [ 0x0 - 0x87000], type: 0x1
mem[1] : [ 0x87000 - 0x88000], type: 0x2
mem[2] : [ 0x88000 - 0x99000], type: 0x1
mem[3] : [ 0x99000 - 0xa0000], type: 0x2
mem[4] : [ 0xa0000 - 0x100000], type: 0x2
mem[5] : [ 0x100000 - 0x100000], type: 0x1
mem[6] : [ 0x207000 - 0x400000], type: 0x1
mem[7] : [ 0x1e00000 - 0x44633000], type: 0x1
mem[8] : [ 0x44633000 - 0x44973000], type: 0x2
mem[9] : [ 0x44973000 - 0x452b8000], type: 0x2
mem[10] : [ 0x452b8000 - 0x45348000], type: 0x3
mem[11] : [ 0x45348000 - 0x45350000], type: 0x4
mem[12] : [ 0x45350000 - 0x45c00000], type: 0x2
mem[13] : [ 0x45c00000 - 0x45f0f000], type: 0x1
mem[14] : [ 0x45f0f000 - 0x45f10000], type: 0x2
mem[15] : [ 0x45f10000 - 0x45f90000], type: 0x2
mem[16] : [ 0x45f90000 - 0x45f91000], type: 0x4
mem[17] : [ 0x45f91000 - 0x45ff7000], type: 0x3
mem[18] : [ 0x45ff7000 - 0x50400000], type: 0x2
mem[19] : [ 0xfed20000 - 0xfed80000], type: 0x2
mem[20] : [ 0xff200000 - 0x100000000], type: 0x2
mem[21] : [ 0x100000000 - 0x4063f5000], type: 0x1
mem[22] : [ 0x4063f5000 - 0x40640a000], type: 0x2
mem[23] : [ 0x40640a000 - 0x4adc0a000], type: 0x2
mem[24] : [ 0x4adc0a000 - 0x4adca9000], type: 0x2
mem[25] : [ 0x4adca9000 - 0x4adcab000], type: 0x1
mem[26] : [ 0x4adcab000 - 0x4adcac000], type: 0x2
mem[27] : [ 0x4adcac000 - 0x4adcad000], type: 0x1
mem[28] : [ 0x4adcad000 - 0x4ae158000], type: 0x2
mem[29] : [ 0x4ae158000 - 0x4ae15a000], type: 0x1
mem[30] : [ 0x4ae15a000 - 0x4ae9aa000], type: 0x2
mem[31] : [ 0x4ae9aa000 - 0x4ae9ba000], type: 0x4
mem[32] : [ 0x4ae9ba000 - 0x4af804000], type: 0x2
mem[33] : [ 0x4af804000 - 0x4af80c000], type: 0x4
mem[34] : [ 0x4af80c000 - 0x4afc00000], type: 0x2

Reply:
LGTM